src/customs/Bus.java

```diff
@@ -1,17 +1,76 @@
 package src.customs;
 
+import src.algorithms.MyBinarySearch;
+import src.algorithms.MySorting;
+
 import java.io.Serializable;
+import java.util.Objects;
 
-public class Bus implements Serializable {
-    private Integer number;
+public class Bus implements Comparable<Bus>, Serializable {
+    private int number;
     private String model;
-    private Integer mileage;
+    private int mileage;
+    private int year;
 
-    public Bus(Integer number, String model, Integer mileage) {
-        this.number = number;
-        this.model = model;
+    public int getNumber() {
+        return number;
+    }
+    public String getModel() {
+        return model;
+    }
+    public long getMileage() {
+        return mileage;
+    }
+    public void setMileage(int mileage) {
         this.mileage = mileage;
     }
+    public int getYear() {
+        return year;
+    }
+    public void setYear(int year) {
+        this.year = year;
+    }
```

Comment:
Методы слишком близком друг к другу. Раздели пробелом, а то читать сложно
```suggestion
    public int getNumber() {
        return number;
    }
    
    public String getModel() {
        return model;
    }
    
    public long getMileage() {
        return mileage;
    }
    
    public void setMileage(int mileage) {
        this.mileage = mileage;
    }
    
    public int getYear() {
        return year;
    }
    
    public void setYear(int year) {
        this.year = year;
    }
```